src/test/rest_utils_test.cpp

```diff
@@ -183,44 +190,44 @@ TEST_F(RestUtilsTest, MakeJsonFromPredictResponse_ColumnOrder) {
     }
 })";
 
-    EXPECT_TRUE(is_in_first_order || is_in_second_order);
+std::string getJsonResponseInSecondOrder(ovms::Order order, const char* secondOrderRow, const char* secondOrderColumn) {
```

Comment:
How does that function differ from `getJsonResponseInFirstOrder` beside variables naming?

Reply:
Similar to getJsonResponseDependsOnOrder as well?